python/doc/examples/probabilistic_modeling/random_vectors/plot_ratio_of_uniforms_exploitation.py

```diff
@@ -0,0 +1,455 @@
+"""
+Use the Ratio of Uniforms algorithm to sample a distribution
+============================================================
+"""
+
+# %%
+# In this example, we illustrate in dimension :math:`\inputDim = 1` the ratio of uniforms algorithm  to sample a distribution which
+# probability density function is known up to a normalization factor.
+#
+# Consider a scalar distribution whose
+# probability density function :math:`p` is known up to a normalization factor:
+# let :math:`f: \Rset \rightarrow \Rset^+` be a function such that
+# :math:`p(x) = cf(x)` for all :math:`x \in \Rset` with :math:`c \in \Rset^+_*`.
+#
+# Let :math:`r \in \Rset^+_*`. Let  :math:`A_{f,r}` be the domain defined by:
+#
+# .. math::
+#
+#    A_{f,r} = \left\{ (u, v) \in \Rset^2 \, |\, 0 \leq u \leq f\left(\dfrac{v}{u^r}\right)^{\frac{1}{1+r}}\right \}
+#
+# The Lebesgue measure of :math:`A_{f,r}`  is equal to :math:`\dfrac{1}{c(1+r)}`.
+#
+# Let :math:`(U, V)` be a random variable uniformly distributed on :math:`A_{f,r}`.
```

Comment:
a set of random variables or a random variable vector ?

Reply:
random vector.....